extensions/todoist/src/menu-bar.tsx

```diff
@@ -204,6 +218,36 @@ const TodayView = ({ tasks }: TaskViewProps) => {
   }
 };
 
+const FilterView = ({ tasks }: TaskViewProps) => {
+  const { data: stats } = useCachedPromise(() => getProductivityStats());
+
+  const todayStats = stats?.days_items.find((d) => d.date === format(Date.now(), "yyyy-MM-dd"));
+  const completedToday = todayStats?.total_completed ?? 0;
+
+  if (tasks.length > 0) {
+    return (
+      <MenuBarExtra.Section title={"Filtered tasks"} key={"filterTasks"}>
+        {tasks.map((task) => {
+          return <MenuBarTask key={task.id} task={task} />;
+        })}
+      </MenuBarExtra.Section>
+    );
+  }
+
+  if (completedToday > 0) {
+    return (
+      <MenuBarExtra.Section title={"No tasks matching filter"} key={"filterTasks"}>
+        <MenuBarExtra.Item
+          title={`Congrats! You've completed ${completedToday} ${completedToday === 1 ? "task" : "tasks"} today.`}
+          icon="🎉"
+        />
+      </MenuBarExtra.Section>
+    );
+  } else {
```

Comment:
I don't think this screen is needed when using filters. The completed tasks count is mostly used for the today's view empty screen.

Reply:
I think it's still nice. In my case, I use filters to show the tasks I need to do today but filter by project and label instead of Due dates (as due dates I use mostly to mark when should something be completed by). So having the nice congrats message does make me happy

Reply:
removed then :D